### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by jbampton**
----
📚 Documentation preview 📚: https://cloudstack-documentation--373.org.readthedocs.build/en/373/

### CODE REVIEW

File: source/conceptsandterminology/locale/pot/developer_guide.pot [hunk 1]
```diff
@@ -443,7 +443,7 @@ msgstr ""
...
... used to implement the default web UI. This API is not a standard like `OGF OCCI <http://www.ogf.org/gf/group_info/view.php?group=occi-wg>`__ or `DMTF CIMI <http://dmtf.org/standards/cloud>`__ but is easy to learn. Mapping exists between the AWS API and the CloudStack API as will be seen in the next section. Recently a Google Compute Engine interface was also developed that maps the GCE REST API to the CloudStack API described here. The API `docs <https://cloudstack.apache.org/api.html>`__ are a good start to learn the extent of the API. Multiple clients exist on `GitHub <https://github.com/search?q=cloudstack+client&ref=cmdform>`__ to use this API, you should be able to find one in your favorite language. The reference documentation for the API and changes that might occur from version to version is availble `on-line <http://cloudstack.apache.org/docs/en-US/Apache_CloudStack/4.1.1/html/Developers_Guide/index.html>`__. This short section is aimed at providing a quick summary to give you a base understanding of how to use this API. As a quick start, a good way to explore the API is to navigate the dashboard with a firebug console (or similar developer console) to study the queries."
```
Comment (DaanHoogland) [inline source/conceptsandterminology/locale/pot/developer_guide.pot:446]: what has changed here?

Comment (jbampton) [reply source/conceptsandterminology/locale/pot/developer_guide.pot:446]: The sentence that starts with:

Multiple clients exist on `github

changed to:

Multiple clients exist on `GitHub

Comment (DaanHoogland) [reply source/conceptsandterminology/locale/pot/developer_guide.pot:446]: ok, so spoiled by the highlighting on githup i am

**Review by DaanHoogland — APPROVED** [review]

File: source/conceptsandterminology/locale/pot/developer_guide.pot (hunk 1 above)
Comment (DaanHoogland) [reply source/conceptsandterminology/locale/pot/developer_guide.pot:446]: and by autospelolcorrectors

**Pull request merged**